examples/dial/psk/dial_psk.rs

```diff
@@ -1,16 +1,16 @@
-use anyhow::Result;
 use clap::{App, AppSettings, Arg};
 use std::io::Write;
 use std::sync::Arc;
 use tokio::net::UdpSocket;
 use util::Conn;
 use webrtc_dtls::cipher_suite::CipherSuiteId;
+use webrtc_dtls::Error;
 use webrtc_dtls::{config::*, conn::DTLSConn};
 
 // cargo run --example dial_psk -- --server 127.0.0.1:4444
 
 #[tokio::main]
-async fn main() -> Result<()> {
+async fn main() -> Result<(), Error> {
```

Comment:
why not add use "use webrtc_dtls::error::Result" to avoid this change? 

I saw many such changes which can be avoided.

Reply:
Got it